Podcast/ProfileHeaderView.swift

```diff
@@ -0,0 +1,254 @@
+        subscriptions.allowsSelection = false
+        
+//        bottomButton3 = makeBotBarButton(3)
+//        bottomBar.addSubview(bottomButton3)
+        
+        self.addSubview(bottomBar)
+        self.addSubview(topBar)
+        self.addSubview(subsHeader)
+        self.addSubview(subscriptions)
+        self.addSubview(profileArea)
+    }
+    
+    let proImgWidth:CGFloat = 60
+    let proImgY: CGFloat = 28.5
+    let padding: CGFloat = 12
+    let labelHeight: CGFloat = 19
+    let labelSpacing: CGFloat = 2.5
+    
+    override func layoutSubviews() {
+        // Width for bottom bar buttons
+        let bButtonWidth:CGFloat = self.frame.size.width/numBottomBarButtons
+        bottomButton1.frame = CGRect(x: 0, y: 0, width: bButtonWidth, height: bottomBarHeight)
+        bottomButton2.frame = CGRect(x: bButtonWidth, y: 0, width: bButtonWidth, height: bottomBarHeight)
+//        bottomButton3.frame = CGRect(x: 2*bButtonWidth, y: 0, width: bButtonWidth, height: bottomBarHeight)
+        
+        let vertBarPadding: CGFloat = 12
+        let verticalBar1 = UIView(frame: CGRect(x: bButtonWidth, y: vertBarPadding, width: 1, height: bottomBarHeight-2*vertBarPadding))
+        verticalBar1.backgroundColor = UIColor.podcastGray
+//        let verticalBar2 = UIView(frame: CGRect(x: 2*bButtonWidth, y: vertBarPadding, width: 1, height: bottomBarHeight-2*vertBarPadding))
+//        verticalBar2.backgroundColor = UIColor.podcastGray
+        bottomBar.addSubview(verticalBar1)
+//        bottomBar.addSubview(verticalBar2)
+        
+        let labelWidth: CGFloat = self.frame.size.width-2*padding
+        let usernameY = proImgY+proImgWidth+padding+labelHeight+labelSpacing
+        profileArea.frame = CGRect(x: 0, y: statusBarHeight, width: self.frame.size.width, height: bottomBarDist-statusBarHeight)
+        profileImage.frame = CGRect(x: (self.frame.size.width-proImgWidth)/2, y: proImgY, width: proImgWidth, height: proImgWidth)
+        profileImage.layer.cornerRadius = proImgWidth/2.0
+        nameLabel.frame = CGRect(x: padding, y: proImgY+proImgWidth+padding, width: labelWidth, height: labelHeight)
+        usernameLabel.frame = CGRect(x: padding, y: usernameY, width: labelWidth, height: labelHeight)
+        followButton.frame = CGRect(x: (self.frame.size.width-150)/2, y: usernameY+labelHeight+padding, width: 150, height: 29)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func canSeeFollowButton() -> Bool {
+        if let user = user {
+            return !user.isMe
+        }
+        return true
+    }
+    
+    @objc func animateByYOffset(_ yOffset: CGFloat) {
+        let usernameY = proImgY+proImgWidth+padding+labelHeight+labelSpacing
+        print(yOffset)
+        if yOffset <= 0 {
+            profileImage.alpha = 1
+        } else if yOffset >= (proImgY+proImgWidth/2) {
```

Comment:
`(proImgY+proImgWidth/2)` is also used in line 195, so let's store it in a variable with a specific name so we know what this offset refers to